dhis-2/dhis-api/src/main/java/org/hisp/dhis/merge/MergeType.java

```diff
@@ -27,16 +27,38 @@
  */
 package org.hisp.dhis.merge;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+import org.hisp.dhis.category.CategoryOption;
+import org.hisp.dhis.common.IdentifiableObject;
+import org.hisp.dhis.dataelement.DataElement;
+import org.hisp.dhis.indicator.Indicator;
+import org.hisp.dhis.indicator.IndicatorType;
+
 /**
  * Enum for merge type.
  *
  * @author david mackessy
  */
 public enum MergeType {
-  ORG_UNIT,
```

Comment:
org unit merge was implemented separately & needs to be refactored so it uses the same merge flow as the others. Removing this has no affect as it was never used.